conan_provider.cmake

```diff
@@ -14,15 +33,25 @@ function(detect_os OS OS_VERSION)
         else()
             set(${OS} ${CMAKE_SYSTEM_NAME} PARENT_SCOPE)
         endif()
+        if(ANDROID)
+            string(REPLACE "-" ";" PLATFORM_LIST ${ANDROID_PLATFORM})
+            list(GET PLATFORM_LIST 1 _OS_API_LEVEL)
+            message(STATUS "CMake-Conan: android_platform=${ANDROID_PLATFORM}")
+            set(${OS_API_LEVEL} ${_OS_API_LEVEL} PARENT_SCOPE)
+        endif()
     endif()
 endfunction()
 
 
-function(detect_cxx_standard CXX_STANDARD)
+function(detect_cxx CXX_STANDARD LIB_CXX)
```

Comment:
LIBCXX is a different think than the CXX_STANDARD, I think I'd create a ``detect_lib_cxx`` for it.